scripts/builtin/l2svm.dml

```diff
@@ -112,32 +119,37 @@ Double epsilon = 0.001, Double lambda = 1, Integer maxiterations = 100, Boolean
       step_sz = step_sz - g/h
       continue1 = (g*g/h >= epsilon)
     }
-  
+
     #update weights
     w = w + step_sz*s
     Xw = Xw + step_sz*Xd
-  
+
     out = 1 - Y * Xw
     sv = (out > 0)
     out = sv * out
     obj = 0.5 * sum(out * out) + lambda/2 * sum(w * w)
     g_new = t(X) %*% (out * Y) - lambda * w
-  
-    
-    if(verbose) print("Iter, Obj "+ iter + ", "+obj)
-  
+
+    if(verbose) {
+      if(column_id != -1){
+        print("Iter:" + toString(iter) + ", Col:" + column_id + ", Obj:" + obj)
+      } 
+      else {
+        print("Iter:" + toString(iter) + ", Obj:" + obj)
+      }
+    } 
+
     tmp = sum(s * g_old)
     if(step_sz*tmp < epsilon*obj){
       continue = 0
     }
```

Comment:
Somehow I missed this on the initial commit, but this can become: `continue = (step_sz*tmp >= epsilon*obj & sum(s^2) != 0);`